roles/lnls-ans-role-update-kernel/tasks/update-kernel.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Update kernel
+  package:
+    name: "{{ item }}"
+    state: present
+  with_items: "{{ update_kernel }}"
```

Comment:
We should add the `until` and `retries` options, just as we do for other roles installing packages.

Reply:
done

> We should add the `until` , `retries`, `delay` options, just as we do for other roles installing packages. Also, `become: true` might be important as well if we are not running with privileges enabled Like this:
> 
> ```
>   become: true
>   register: apt_result
>   until: apt_result is success
>   retries: 5
>   delay: 2
> ```

done